ledger-core/v2/insolar/payload/vcallrequestflags.go

```diff
@@ -53,10 +51,7 @@ const (
 )
 
 func (f CallRequestFlags) WithSendResultFull(t SendResultFullFlag) CallRequestFlags {
-	if t == 0 {
-		panic(throw.IllegalValue())
-	}
-	if t > sendResultFullFlagCount {
+	if t > bitSendResultFullFlagCount {
```

Comment:
TWIMC - this looks awful. We compare SendResultFullFlag (type '**byte**') with the **number of bits** and depending on the result doing something. I wish we had better variable names and types, but apparently this is a subject for another PR.

Reply:
just used already approved naming https://github.com/insolar/assured-ledger/blob/master/ledger-core/v2/insolar/payload/vcallflags.go#L31

we can plan a new task about renaming and I can take it